src/applications/ivc-champva/10-7959C/chapters/medicareInformation.js

```diff
@@ -5,411 +5,383 @@ import {
   titleSchema,
   currentOrPastDateUI,
   currentOrPastDateSchema,
+  radioUI,
+  radioSchema,
+  yesNoUI,
+  yesNoSchema,
 } from 'platform/forms-system/src/js/web-component-patterns';
-import { applicantListSchema, requiredFiles } from '../config/constants';
-import { applicantWording } from '../../shared/utilities';
-import ApplicantField from '../../shared/components/applicantLists/ApplicantField';
-import { isRequiredFile } from '../helpers/utilities';
+import { requiredFiles } from '../config/constants';
+import { isRequiredFile, nameWording } from '../helpers/utilities';
 import { fileUploadUi as fileUploadUI } from '../../shared/components/fileUploads/upload';
 import {
   fileWithMetadataSchema,
   fileUploadBlurb,
 } from '../../shared/components/fileUploads/attachments';
 
+// TODO put in /shared
+const additionalFilesHint =
+  'Depending on your response, you may need to submit additional documents with this application.';
+
 export const blankSchema = { type: 'object', properties: {} };
 
-// Used with a custom page
 export const applicantHasMedicareABSchema = {
   uiSchema: {
-    applicants: { items: {} },
-  },
-  schema: applicantListSchema([], {
+    ...titleUI(({ formData }) => `${nameWording(formData)} Medicare status`),
     applicantMedicareStatus: {
-      type: 'object',
-      properties: {
-        enrollment: { type: 'string' },
-        otherEnrollment: { type: 'string' },
-      },
+      ...yesNoUI({
+        updateUiSchema: formData => {
+          let title = 'Do you have Medicare Parts A & B to add or update?';
+          if (formData?.certifierRole !== 'applicant') {
+            title = `Does ${nameWording(
+              formData,
+              false,
+            )} have Medicare Parts A & B to add or update?`;
+          }
+          return {
+            'ui:title': title,
+            'ui:options': { hint: additionalFilesHint },
+          };
+        },
+      }),
     },
-  }),
+  },
+  schema: {
+    type: 'object',
+    required: ['applicantMedicareStatus'],
+    properties: {
+      applicantMedicareStatus: yesNoSchema,
+    },
+  },
 };
```

Comment:
All the changes in this file are pretty much just like this:
- Remove the uiSchema `applicants: { items: {...}}` property
- Remove the boilerplate from `applicantListSchema`
- Add in `yesNoUI` (or whatever other UI we need) - many of these configs previously relied on custom pages, but that will no longer be the case outside of the list loop.